src/Mobs/Monster.cpp

```diff
@@ -367,6 +421,39 @@ void cMonster::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 
 
+void cMonster::CalcLeashActions()
+{
+	// This mob just spotted in the world and [m_LeashToPos not null] shows that should be leashed to a leash knot at m_LeashToPos.
+	// This keeps trying until knot is found
```

Comment:
Add a note that the leash knot may be in a different chunk that needn't be loaded, to make it clear why this can happen.